.circleci/config.yml

```diff
@@ -3,6 +3,14 @@ version: 2.1
 orbs:
   release-management: salesforce/npm-release-management@4
 
+parameters:
```

Comment:
Pipeline parameters used to run only NUT tests. Code that triggers/monitors will set `run-auto-workflows` to false and `run-just-nuts` to true. With all workflows guarded with one of these vars it will control which run when pipelines run, with the goal of only running NUTs tests when triggering just nuts from last step of sfdx-cli release pipeline.